src/global.h

```diff
@@ -119,28 +119,30 @@ class global_scope_t : public noncopyable, public scope_t
   }
 
   void show_version_info(std::ostream& out) {
-    out <<
-      "Ledger " << Ledger_VERSION_MAJOR << '.' << Ledger_VERSION_MINOR << '.'
-                << Ledger_VERSION_PATCH;
-    if (Ledger_VERSION_PRERELEASE != 0)
-      out << Ledger_VERSION_PRERELEASE;
-    if (Ledger_VERSION_DATE != 0)
-      out << '-' << Ledger_VERSION_DATE;
-    out << _(", the command-line accounting tool");
-    out << _("\nwith");
-#if !HAVE_GPGME
-    out << _("out");
+    std::string version  = Ledger_VERSION;
+    if (strlen(Ledger_VERSION_PRERELEASE))
+      version += "-" Ledger_VERSION_PRERELEASE;
+    out
+      << _f("Ledger %1%, the command-line accounting tool") % version
+      << std::endl
+#if HAVE_GPGME
+      << _("with support for gpg encrypted journals")
+#else
+      << _("without support for gpg encrypted journals")
 #endif
-    out << _(" support for gpg encrypted journals and with");
-#if !HAVE_BOOST_PYTHON
-    out <<_("out");
+      << " "
+#if HAVE_BOOST_PYTHON
+      << _("and with Python support")
+#else
+      << _("and without Python support")
```

Comment:
I don't like how the message for Ledger's gpg and Python capabilities is pieced togeher.

Neovim's output with the `-v` option includes a "Features" line, e.g. `Features: +acl +iconv +tui`. For Ledger this could be: `Features: +gpg +py` is this an improvement over the current output?